26/ex26.py

```diff
@@ -0,0 +1,92 @@
+def break_words(stuff):
+    """This function will break up words for us."""
+    words = stuff.split(' ')
+    return words
+
+def sort_words(words):
+    """Sorts the words."""
+    return sorted(words)
+
+def print_first_word(words):
+    """Prints the first word after popping it off."""
+    word = words.pop(0)
+    print word
+
+def print_last_word(words):
+    """Prints the last word after popping it off."""
+    word = words.pop(-1)
+    print word
+
+def sort_sentence(sentence):
+    """Takes in a full sentence and returns the sorted words."""
```

Comment:
If you have a docstring (a comment for a function) that's one line then you should format the end of the string on a separate line, e.g.
```python
def hello():
    """Say hello.
    """
    return "Hello!"
```